liberty-remote/JakartaEE9_README.md

```diff
@@ -124,6 +136,11 @@ xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/a
 			<artifactId>arquillian-liberty-remote-jakarta</artifactId>
 			<version>2.0.0</version>
 		</dependency>
+		<dependency>
+			<groupId>io.openliberty.arquillian</groupId>
+			<artifactId>arquillian-liberty-rest-protocol-jakarta</artifactId>
+			<version>2.0.0</version>
```

Comment:
version `2.1.0` and make above dependency consistent

Reply:
I have updated this in the PR.  Things will be updated again when there is a new Alpha release of Arquillian.